src/index.tsx

```diff
@@ -8,22 +8,24 @@ import {
   createBrowserRouter,
   createRoutesFromElements,
 } from 'react-router-dom';
-import { ToastContainer, toast, Slide } from 'react-toastify';
+import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import { HomePage } from './pages/HomePage/HomePage';
 import { LikePage } from './pages/LikePage/LikePage';
 import { BookmarkPage } from './pages/BookmarkPage/BookmarkPage';
 import { messages } from './data/messages';
 import { persons } from './data/persons';
-import MessageStructure from './model/Message';
+import MessageStructure from './types/Message.ts';
 import dayjs from 'dayjs';
 import { getRandomPerson } from './helpers/getRandomPerson.ts';
 import { SearchProvider } from './context/SearchContext.tsx';
 import { UserContext, UserProvider } from './context/UserContext.tsx';
+import { useToast } from './hooks/useToast.tsx';
 
 const Main = () => {
   const [messagesData, setMessagesData] = useState<MessageStructure[]>([]);
```

Comment:
Když tak na to koukám, i když je ten souboru už mnohem kratší, možná by si messages zasloužily taky vlastní kontext.

Předáváš je do hodně komponent (hluboký props drilling) a máš k nim hodně funkcí, např. přidávání nové zprávy, když by to šlo z kontextu, bys mohla zavolat rovnou z Post komponentu a nemusela bys to odtud předávat do Homepage a z Homepage do Post komponentu.